airflow/jobs/backfill_job.py

```diff
@@ -471,6 +476,13 @@ def _per_task_process(key, ti, session=None):
                             ti_status.running.pop(key)
                         # Reset the failed task in backfill to scheduled state
                         ti.set_state(State.SCHEDULED, session=session)
+                elif self.rerun_succeeded_tasks and ti.state == State.SUCCESS:
+                    # Rerun succeeded tasks
+                    self.log.info("Task instance {ti} with state {state}".format(ti=ti, state=ti.state))
```

Comment:
```suggestion
                    self.log.info("Task instance %s with state %s", ti, ti.state)
```
For some reasons, we don't use `format` in log messages. We prefer using `%s`. Also, should we make the log to convey a good message, like, Task instance with state ... succeeded. rerunning task? what do you think?

Reply:
Yes, it will make the feature clearer to users.

Reply:
I have updated log message in recent commit